bin/configs/python-lazyImports.yaml

```diff
@@ -0,0 +1,16 @@
+generatorName: python
+outputDir: samples/openapi3/client/petstore/python-lazyImports
```

Comment:
please add the new output folder to the workflow

https://github.com/OpenAPITools/openapi-generator/blob/master/.github/workflows/samples-python-petstore.yaml